.rubocop.yml

```diff
@@ -117,26 +117,18 @@ Rails/DynamicFindBy:
   Exclude:
     - "app/sample_manifest_excel/**/*"
     - "test/app/sample_manifest_excel/**/*"
-  Whitelist:
+  AllowedMethods:
```

Comment:
Rubocop itself now complains about whitelist, but doesn't check its own files. I'm not quite sure why it doesn't have consistency between AllowedMethods in some cases and IgnoredMethods in others.